Formula/pacparser.rb

```diff
@@ -0,0 +1,28 @@
+class Pacparser < Formula
+  desc "Library to parse proxy auto-config (PAC) files"
+  homepage "https://github.com/pacparser/pacparser"
+  url "https://github.com/pacparser/pacparser/archive/1.3.6.tar.gz"
```

Comment:
Been a bit tied up with other projects the last few days but I will get this updated hopefully this week.  I haven't forgotten about it yet ;p